package/fusemodule/package

```diff
@@ -0,0 +1,73 @@
+e!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(fusemodule)
```

Comment:
I'd recommend just calling this fuse instead of fusemodule.

Reply:
Done!